services/nublado2/values-int.yaml

```diff
@@ -16,6 +16,7 @@ nublado2:
 
   config:
     base_url: "https://lsst-lsp-int.ncsa.illinois.edu"
+    butler_secret_path: "secret/k8s_operator/lsst-lsp-int.ncsa.illinois.edu/butler-secret"
```

Comment:
I don't think these credentials are meaningful for NCSA, are they? The PostgreSQL server at least wouldn't be reachable.

Reply:
They are not meaningful.  However, rather than put a bunch of conditional logic in the resource manager, I'm just putting dummy secrets in all environments.